shell/platform/fuchsia/flutter/BUILD.gn

```diff
@@ -452,3 +453,134 @@ fuchsia_archive("flutter_runner_tzdata_tests") {
     },
   ]
 }
+
+executable("flutter_runner_scenic_unittests") {
```

Comment:
IIRC, because of dependencies on session_connection, Kaushik and I made flutter_runner_scenic_tests a separate target as it was not possible to do so otherwise.